src/main/java/org/minions/devfund/alejandro/WhackAMole.java

```diff
@@ -0,0 +1,132 @@
+package org.minions.devfund.alejandro;
+
+/**
+ * Class for whack a mole.
+ * It contains three integer instance variables called score, molesLeft, and attemptsLeft.
+ * Make one more instance variable called moleGrid which is a 2-dimensional array of chars.
+ */
+public class WhackAMole {
+
+    private int score;
+    private int molesLeft;
+    private int attemptsLeft;
+    private char[][] moleGrid;
+    private StringBuilder auxiliaryBuilder = new StringBuilder();
```

Comment:
Declare as attribute of class will have issues in the future.
Please declare as local variable inside method.